assets/js/admin-script.js

```diff
@@ -270,22 +270,23 @@ jQuery( function( $ ) {
 	$( document ).on( 'change', '#wpo-wcpdf-settings input[type="radio"], #wpo-wcpdf-settings select', function( event ) {
 		if ( ! settingIsExcludedForPreview( $( this ).attr( 'name' ) ) ) {
 			if ( ! event.isTrigger ) { // exclude programmatic triggers that aren't actually changing anything
-				settingsChanged();
+				triggerPreview();
 			}
 		}
 	} );
 
 	// Preview on header logo change
 	$( document.body ).on( 'wpo-wcpdf-media-upload-setting-updated', function( event, $input ) {
-		settingsChanged();
+		triggerPreview();
 	} );
 	$( document ).on( 'click', '.wpo_remove_image_button', function( event ) {
-		settingsChanged();
+		triggerPreview();
 	} );
 
 	// Custom trigger to signify settings have changed (will show save button and refresh preview)
 	$( document ).on( 'wpo-wcpdf-settings-changed', function( event, delay ) { 
-		settingsChanged( delay );
+		showSaveBtn();
+		triggerPreview( delay );
```

Comment:
we still need this!

Reply:
I've combined everything in the `settingChanged()` function.